man/tpm2_createprimary.1.md

```diff
@@ -30,7 +30,7 @@ will create and load a Primary Object. The sensitive area is not returned.
       * **n** for **TPM_RH_NULL**
       * **`<num>`** where a raw number can be used.
 
-  * **-P**, **--auth-hierarchy**=_HIERARCHY\_\_AUTH\_VALUE_:
```

Comment:
This was correct AFAICT.

Reply:
Yeah it was, I was way to greedy in my sed find/replace.